win32/src/win32evtlog.i

```diff
@@ -511,6 +512,113 @@ static PyObject *PyEvtOpenChannelEnum(PyObject *self, PyObject *args, PyObject *
 }
 PyCFunction pfnPyEvtOpenChannelEnum = (PyCFunction) PyEvtOpenChannelEnum;
 
+
+//Helper function to convert a list of strings double zero terminated
```

Comment:
Could you please add spaces between the slash and the first letter of the comment, and remove the spaces after parens? (throughout the entire patch, not just here)